test/redis/store/marshalling_test.rb

```diff
@@ -23,6 +23,12 @@ def teardown
     @store.get("rabbit").must_equal(@white_rabbit)
   end
 
+  it "marshals on multi set" do
+    @store.mset "rabbit", @white_rabbit, "rabbit2", @rabbit
```

Comment:
Yeh, no sweat. I'll update when I get time later tonight. After I opened the PR, I also noticed that `Redis::Store::Namespace` has code for `get`, `mget`, and friends. Presumably, I should to add code for `mset` as well, right?